packages/flutterfire_starter/hooks/post_gen.dart

```diff
@@ -34,6 +38,24 @@ Future<void> _installDependencies(HookContext context) async {
     ['pub', 'add', 'firebase_core'],
     workingDirectory: './$appName',
   );
+
+  final varsPlugins = (context.vars['plugins'] as List<dynamic>).cast<String>();
+  final varsPluginsName = varsPlugins.map((e) => e.split(' ')[0]).toList();
+  if (varsPluginsName.contains('Analytics')) {
+    await Process.run(
+      'flutter',
+      ['pub', 'add', 'firebase_analytics'],
+      workingDirectory: './$appName',
+    );
+  }
+  if (varsPlugins.contains('Analytics with GoRouter')) {
```

Comment:
possibly make a note that this is the value pulled from the brick.yaml (I think) so it's understood where it is coming from.

Reply:
I've added that